.gitignore

```diff
@@ -1,3 +1,5 @@
+*.orig
+*.rej
```

Comment:
Then I prefer not to apply the patch.  It adds no value to logrotate.  Moreover, the presence of ```*.orig``` and ```*.rej``` usually indicates that some patches did not apply properly.  Hiding it can be dangerous in certain situations.